container/mms_config_image_classification.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+MMS_NUM_WORKER=1
+MMS_NUM_THREAD_PER_WORKER=`nproc`
```

Comment:
What is the number of workers setting before this change?

Reply:
https://github.com/neo-ai/neo-ai-dlr/blob/7d56499a210dcd0bea077a52914576d6fb721e89/container/dockerd-entrypoint.sh#L5-L6